api/src/pcapi/core/finance/api.py

```diff
@@ -1288,6 +1299,7 @@ def _generate_bank_accounts_file(cutoff: datetime.datetime) -> pathlib.Path:
     )
 
     row_formatter = lambda row: (
+        ",".join(sorted(str(venue_id) for venue_id in row.venue_ids)),
```

Comment:
- Un espace serait bienvenue pour aérer la lecture.
- Je ne suis pas fan de trier par ID converti en chaîne de caractères, car on aura par exemple : `"1, 10, 100, 2, 20, 200"`, qui n'est pas un ordre naturel.

```suggestion
        ", ".join(str(venue_id) for venue_id in sorted(row.venue_ids)),
```

Sinon, ajouter `offerers_models.VenueBankAccountLink.venueId` dans le `order_by` permettrait-il de s'assurer directement de l'ordre ?

Reply:
Euh oui, mon sorted est mal placé
Et bien vu pour l'espace

Reply:
Ne t'embête pas, ça marchera très bien avec le `sorted` :)